packages/vscode-extension/src/debug/teamsfxTaskHandler.ts

```diff
@@ -505,7 +505,7 @@ export function terminateAllRunningTeamsfxTasks(): void {
     }
   }
   allRunningTeamsfxTasks.clear();
-  LocalTunnelTaskTerminal.stopAll();
+  NgrokTunnelTaskTerminal.stopAll();
```

Comment:
Do we need to also stop all dev tunnel tasks?

Reply:
Use a new URL each time debug is due to security issue. Fixed URLs can be easily predicted and exploited by malicious individuals.

PS: The dev tunnel team will support a shorter lifetime tunnel in the furture.
> Lastly, this not implemented but we have briefly discussed the idea of providing an option to [only connect anonymously for a short period](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2Fbasis-planning%2Fissues%2F699&data=05%7C01%7CXiaofu.Huang%40microsoft.com%7C9d2674aef7064633271f08db108f902b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638121981043649814%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=5avZAJqPw7nxqc6GLwoGPiHKAdRvzXdsm%2BErI%2Fabosw%3D&reserved=0) (e.g. 1hr, 1d) or until a certain time in the future. Then anonymous access would automatically be removed by our service.

Reply:
Thx! I updated it to `BaseTunnelTaskTerminal`